paasta_tools/contrib/mock_patch_checker.py

```diff
@@ -20,32 +19,48 @@ def check_files(self, files):
     def check_file(self, filename):
         self.current_filename = filename
         try:
-            ast = compiler.parseFile(filename)
-        except SyntaxError, error:
-            print "SyntaxError on file %s:%d" % (filename, error.lineno)
+            with open(filename, 'r') as fd:
+                try:
+                    file_ast = ast.parse(fd.read())
+                except SyntaxError as error:
+                    print "SyntaxError on file %s:%d" % (filename, error.lineno)
+                    return
+        except IOError:
+            print "Error opening filename: %s" % filename
             return
-        compiler.walk(ast, self)
         self.init_module_imports()
+        self.visit(file_ast)
 
-    def visitImport(self, node):
-        if (name for name in node.names if 'mock' in name):
+    def CallUsesMockPatch(self, node):
```

Comment:
this should be `def call_uses_mock_patch(self, node):` or even `def _call_uses_mock_patch(...):`